### ISSUE TITLE
Test the model on Videos

### ISSUE THREAD

**Issue by AditTuf** [ZhenboSong/mono_velocity#2]
Hi ZhenboSong . Let me thank you for making your codebase public .Can you please tell me how can I test this model on some another video recorded from top of the car driving on a Road ,the same scenario as shown in the demo image

**Comment by Vibhuvan** [comment]
Yes, please look into this at the earliest and try to provide the inference code (so that we can test it on a video/img)
